react-ui/src/components/LogInForm.js

```diff
@@ -13,8 +13,8 @@ class logInForm extends Component {
     super(props);
     this.state = {
       islogged: false,
-      firstName: "",
       type: "",
+      name: "",
```

Comment:
Minor change, but I would recommend renaming it to firstName to increase code readability in the future.